lib/migration.js

```diff
@@ -423,7 +432,7 @@ function mixinMigration(MySQL, mysql) {
         fkSQL = self.getForeignKeySQL(model, newFks);
     }
     if (fkSQL && fkSQL.length) {
-      self.applySqlChanges(model, fkSQL, function(err, result) {
+      self.applySqlChanges(model, [fkSQL.join(', ')], function(err, result) {
```

Comment:
@darknos you just changed it as I was writing the comment. `toString()` it is :)